app/conf/Configuration.scala

```diff
@@ -65,7 +65,7 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
     val credentials: Option[AWSCredentialsProvider] = {
       val provider = new AWSCredentialsProviderChain(
         new ProfileCredentialsProvider("cmsFronts"),
-        new InstanceProfileCredentialsProvider
```

Comment:
This got deprecated see: http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/InstanceProfileCredentialsProvider.html